### PR TITLE
Bird & BirdRegion scrape done 

### PR DESCRIPTION
**Pull request by cassiemak**

### CODE REVIEW

File: lib/tasks/get_birds.rake [hunk 1]
```diff
@@ -26,37 +26,44 @@ namespace :scrape_birds do
...
-      end
+      begin
+        bird_common_name = bird.css('a').attr('title') ? bird.css('a').attr('title').text.squish : ''
+
+        case bird_common_name
+          when "Pacific loon"
+            start = true
+          when "List of birds"
+            start = false
+        end
```
Comment (h8rry) [inline lib/tasks/get_birds.rake:44]: The indentation for case statement should be

``` ruby
case bird_common_name
when "Pacific loon"
  start = true
when "List of birds"
  start = false
end
```

File: lib/tasks/make_region_plants.rake [hunk 2]
```diff
@@ -65,12 +65,12 @@ namespace :make_region_plants do
             Plant.where("scientific_name like ?", "%#{array[3]}%").each do |f|
```
Comment (h8rry) [inline lib/tasks/make_region_plants.rake]: Indentations in this file seem to be off, seen from here: https://github.com/cchchoi1986/hikehk/blob/regional/lib/tasks/make_region_plants.rake

Make sure that the tab size = 2

**Comment by h8rry** [comment]
The rest looks good :)

**Comment by cassiemak** [comment]
Plants is clem's file. Wont touch it.
I'll merge after fixing the indentation.
Thanks!

**Comment by cassiemak** [comment]
I lied. I have fixed the Plant scrape file indention. MErging now.

**Pull request merged**